### DISCUSSION TITLE
Storage space issue?

### DISCUSSION THREAD

**Discussion by otaworks** [keiwando/toonsquid#653] [Q&A]
Hi, everyone! 
I checked my storage space in System settings and found out that Toonsquid currently has over 4gb of storage space, and upon tapping and scrolling through the space in the System settings, I found some files like exported images, among other files. 
How can I clear this huge storage space takeover and reduce the file size? Perhaps I should just uninstall and reinstall the app to clear the space?

**Comment by budaloops** [comment]
This happens to me with Affinity Designer. It’s the app I use a lot. I uninstall and reinstall to fix. Wondering if there’s a better solution too

**Reply by otaworks** [reply]
Thanks! Hopefully ToonSquid 2.0 will bring a fix to this. Even Callipeg has the same issue

**Comment by Cat-Butler** [comment]
Be aware that, like many apps, uninstalling and reinstalling will wipe out everything in the gallery view (all projects, unless exported as .tsproj files to another location outside the ToonSquid folder) - although that's probably where a lot of that storage space is located.
Manually exported .tsproj, .gif, .png, .mp4 files located outside the ToonSquid folder should be safe.

**Comment by keiwando** [comment]
If you are using the [backup feature](https://toonsquid.com/handbook/projects/backup/) (which I would always recommend), then ToonSquid by default keeps three separate copies of your entire project file as backups. If you have older projects that you are not actively working on, you could delete older backups and maybe only keep one or just export the latest project file and save it outside of your iPad somewhere and then delete all of that project's backups.

Also, as @Cat-Butler correctly states, never just delete ToonSquid off of your iPad without at least manually saving all of your projects and brushes, color palettes etc. somewhere else. Deleting the app will delete all of your data.
